ApeWorX/ape-notebook
# chore: update ape core 0.5

### What I did



- Updated ape core to >=0.5.0,<0.6.0: fixes issue with cli where ape doesn't recognize `ape notebook` as a command
- Updated template dependencies for current state in `ApeWorX/project-template`
- Fixed `CONTRIBUTING.md` for a small typo

### How I did it

Changed `setup.py` file.

### How to verify it

1. Install `ape-notebook` plugin pointing to this branch with `eth-ape>=0.5.0,<0.6.0`. 
2. Run `ape notebook`.
3. Check notebook server starts in `./notebook/` directory

### Checklist
